src/connection/server_support.c

```diff
@@ -2741,7 +2741,7 @@ css_process_new_slave (SOCKET master_fd)
 
   auto wp = cubthread::internal_tasks_worker_pool::get_instance ();
   cubthread::get_manager ()->push_task (wp, new_slave_task);
-  // *INDENT-ON*
+  // todo: wait for ha_Server_state to become HA_SERVER_STATE_ACTIVE
```

Comment:
when should this todo be solved? a specific jira issue might be needed if it hasn't one already.